spark-bigquery-connector-common/src/test/resources/acceptance/big_numeric.py

```diff
@@ -0,0 +1,29 @@
+import sys
```

Comment:
We have two sets of .py files. Can we move them to some common resources directory/folder?

Reply:
Done